src/App.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Route, Switch } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import store from './redux/configureStore';
+import Locations from './pages/Locations';
+import Details from './pages/Details';
+import NavBar from './components/Navbar';
+import { getLocations } from './redux/api/api';
+
+import './App.scss';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  componentDidMount() {
+    if (this.props) {
+      // eslint-disable-next-line react/prop-types
```

Comment:
- [ ] Here as well, I would not recommend disabling eslint. instead, try to make the code standard to follow the eslint rules. :slightly_smiling_face:

Reply:
Ok. I solved it already. :+1: